plugins/modules/pf.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Florian Paul Azim Hoberg @gyptazy <gyptazy@gyptazy.ch>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: pf
+version_added: 6.3.0
+short_description: Manage BSD Packet Filter (pf)
+description:
+    - This module manages the BSD Packet Filter (pf).
+options:
+  config:
+    description:
+      - Path to a pf rule set.
+    type: path
+    required: true
+  filter:
+    description:
+      - If state is C(ignore), filters are ignored.
+      - If state is C(filter), only filter rules will be loaded.
+      - If state is C(nat), only nat rules will be loaded.
+      - If state is C(options), only options rules will be loaded.
+    choices: [ 'ignore', 'filter', 'nat', 'options' ]
+    type: str
+    default: ignore
+  state:
+    description:
+      - If state is C(started), packet filter will be started.
+      - If state is C(stopped), packet filter will be stopped.
+      - If state is C(restarted), packet filter will be restarted.
+      - If state is C(reloaded), packet filter will be reloaded/flushed.
+    choices: [ 'started', 'stopped', 'restarted', 'reloaded' ]
+    type: str
+    default: reload
+  dry_run:
+    description:
+      - Run C(pfctl) with dry-run option and provides it in meta output.
+    type: bool
+    default: False
```

Comment:
```suggestion
    default: false
```
YAML booleans should always be `true` or `false`.